### PR TITLE
Fixing typo (were->where)

### PR DESCRIPTION
**Pull request by somsubhojit1988**
Fixing a typo in 2nd paragraph of the section "Create a flame graph with system perf tools"

### CODE REVIEW

File: locale/en/docs/guides/diagnostics-flamegraph.md [hunk 1]
```diff
@@ -26,7 +26,7 @@ For diagnosing production deployments, read these notes: [0x production servers]
 
 The purpose of this guide is to show steps involved in creating a flame graph and keep you in control of each step.
 
-If you want to understand each step better take a look at the sections that follow were we go into more detail.
+If you want to understand each step better take a look at the sections that follow where we go into more detail.
```
Comment (Trott) [inline locale/en/docs/guides/diagnostics-flamegraph.md]: Optional suggestion, but while we're in here, maybe add the missing comma too?:

```suggestion
If you want to understand each step better, take a look at the sections that follow where we go into more detail.
```

**Review by Trott — APPROVED** [review]

**Review by Aissaoui-Ahmed — APPROVED** [review]

**Review by willin — APPROVED** [review]
LGTM

File: locale/en/docs/guides/diagnostics-flamegraph.md [hunk 2]
```diff
@@ -26,7 +26,7 @@ For diagnosing production deployments, read these notes: [0x production servers]
 
 The purpose of this guide is to show steps involved in creating a flame graph and keep you in control of each step.
 
-If you want to understand each step better take a look at the sections that follow were we go into more detail.
+If you want to understand each step better, take a look at the sections that follow where we go into more details.
```
Comment (Trott) [inline locale/en/docs/guides/diagnostics-flamegraph.md]: `detail` is more correct and natural-sounding in my opinion:

```suggestion
If you want to understand each step better, take a look at the sections that follow where we go into more detail.
```

**Comment by ghost** [comment]
@Trott , Sorry forget that 'go into more detail' is a fixed phrase.

**Review by stevemao — APPROVED** [review]

**Pull request merged**